Userland/Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp

```diff
@@ -319,7 +319,7 @@ static SkColorType to_skia_color_type(Gfx::BitmapFormat format)
     case Gfx::BitmapFormat::Invalid:
         return kUnknown_SkColorType;
     case Gfx::BitmapFormat::BGRA8888:
-    case Gfx::BitmapFormat::BGRx8888:
+    case Gfx::BitmapFormat::BGRx8888: // This case requires to set the alpha values to opaque.
```

Comment:
Isn't `kRGB_888x_SkColorType` the right representation for this?

Reply:
Unfortunately Skia has no format which corresponds to `BitmapFormat::BGRx8888`.
With `kRGB_888x_SkColorType` the color bytes are in the wrong order.

Reply:
Perhaps at some point we should align Gfx::BitmapFormat to the libraries we want to use to actually display colors. Skia, Android, etc.

Reply:
The alpha bytes were not initialized by the `BMPImageDecoderPlugin`.
Good idea to fix this earlier in the processing of the bitmap.
I found a quit simple solution by adding `color |= 0xff000000;` in `BMPImageDecoderPlugin`.
I have updated the PR accordingly.